File: Examples/VisualBasic/AsposePdfFacades/Images/DeleteImagesPage.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades

Namespace AsposePDFFacades.Images
    Public Class DeleteImagesPage
        Public Shared Sub Run()
            ' ExStart:DeleteImagesPage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Images()
            ' Open PDF file
            Dim contentEditor As New PdfContentEditor()
            contentEditor.BindPdf(dataDir & Convert.ToString("DeleteImages-Page.pdf"))

            ' Array of images to be deleted
            Dim imageIndex As Integer() = New Integer() {1}

            ' Delete the images from the particular page
            contentEditor.DeleteImage(2, imageIndex)

            ' Save output PDF
            contentEditor.Save(dataDir & Convert.ToString("DeleteImages-Page_out_.pdf"))
            ' ExEnd:DeleteImagesPage           

        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfFacades/Images/DeleteImagesPage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace Aspose.Pdf.Examples.CSharp.AsposePDFFacades.Images
{
    public class DeleteImagesPage
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Images();
            //open PDF file
            PdfContentEditor contentEditor = new PdfContentEditor();
            contentEditor.BindPdf(dataDir+ "DeleteImages-Page.pdf");

            //array of images to be deleted
            int[] imageIndex = new int[] { 1 };

            //delete the images from the particular page
            contentEditor.DeleteImage(2, imageIndex);

            //save output PDF
            contentEditor.Save(dataDir+ "DeleteImages-Page_out.pdf");
 
            
            
        }
    }
}